HIP/hip-904-frictionless-airdrop.md

```diff
@@ -266,194 +454,312 @@ Note: the only change to this API is in the specification, which now allows for
+Below are a some example scenarios to highlight how the transaction record will expose state changes
+across `TokenAirdrop`, `TokenClaim` and `TokenCancel`.
 
 Scenario 1: If Alice airdrops with the following transfer breakout
 
 - 5 FTs of token T1 to Bob, where Bob is already associated with T1
 - NFT serial 1 & 2 of T2 to Bob who is associated with token T2
-- 10 FTs of token T3 to Carol, where Carol is not associated with T3 but has available association slots
-- NFT serials 3 & 4 of T4 to Carol, where Carol is not associated with T4 but has available association slots
-- 15 FTs of token T5 to Dave, where Dave is not associated with T5 and has no available association slots
-- NFT serial 5 & 6 of T6 to Dave, where Dave is not associated with T6 and has no available association slots
-- 20 FTs of token T7 to Eric, where Eric is not associated with T7 and has no available association slots
-- NFT serials 7 & 8 of T8 to Eric, where Eric is not associated with T8 and has no available association slots
+- 10 FTs of token T3 to Carol, where Carol is not associated with T3 but has available
+  association slots
+- NFT serials 3 & 4 of T4 to Carol, where Carol is not associated with T4 but has available
+  association slots
+- 10 FTs of token T5 to Dave, where Dave is not associated with T5 and has no available
+  association slots
+- 5 FTs of token T5 to Dave, where Dave is not associated with T5 and has no available
+  association slots
+- NFT serial 5 & 6 of T6 to Dave, where Dave is not associated with T6 and has no available
+  association slots
+- 20 FTs of token T7 to Eric, where Eric is not associated with T7 and has no available
+  association slots
+- NFT serials 7 & 8 of T8 to Eric, where Eric is not associated with T8 and has no available
+  association slots
 
 The record file emitted from a consensus node will contain
 
 `tokenTransferLists` containing 
 
-- Alice debit of 5 T1 tokens → Bob  credit of 5 T1 tokens
+- Alice debit of 5 T1 tokens → Bob credit of 5 T1 tokens
 - Alice debit of NFT serials 1 & 2 of token T2 → Bob credit of NFT serials 1 & 2 of token T2
-- Alice debit of 10 T3 tokens → Carol  credit of 10 T3 tokens
-- Alice debit of NFT serials 3 & 4 of token T4 → Carol credit of NFT serials 1 & 2 of token T4
+- Alice debit of 10 T3 tokens → Carol credit of 10 T3 tokens
+- Alice debit of NFT serials 3 & 4 of token T4 → Carol credit of NFT serials 3 & 4 of token T4
 
 `automatic_token_associations` containing
 
-- Carols automatic association with token T3
-- Carols automatic association with token T4
+- Carol's automatic association with token T3
+- Carol's automatic association with token T4
 
 `pending_airdrops` containing
 
-- Alices intent to transfer 15 T5 tokens to Dave
-- Alices intent to transfer NFT serials 5 & 6 of token T6 to Dave
+- Alice intent to transfer 15 T5 tokens to Dave
+- Alice intent to transfer NFT serials 5 & 6 of token T6 to Dave
```

Comment:
Aren't we missing Eric's pending airdrops?

Reply:
Will add outside of this PR, thanks